test/integration/graphql/event_integration_test.rb

```diff
@@ -46,15 +42,11 @@ class GraphQLEventTest < ActionDispatch::IntegrationTest
     result = PlaceCalSchema.execute(query_string)
     refute_field result, 'errors'
 
-    data = result['data']
-
-    assert data.has_key?('eventConnection'), 'result is missing key `eventConnection`'
-    connection = data['eventConnection']
-
-    assert connection.has_key?('edges')
-    edges = connection['edges']
+    data = assert_field result, 'data'
```

Comment:
see earlier comment about parenthesis

Reply:
As we talked on the Discord, while I agree with the other cases, in common usage of `assert_*` functions like `assert_field`, `assert_select`, etc. it is generally done without brackets (They seem to be treated like macros tbh).